src/Model/Photo.php

```diff
@@ -596,6 +596,25 @@ public static function newResource()
 		return System::createGUID(32, false);
 	}
 
+	/**
+	 * Extracts the rid from a local photo URI
+	 *
+	 * @param string $image_uri The URI of the photo
+	 * @return string The rid of the photo, or an empty string if the URI is not local
+	 */
+	public static function ridFromURI($image_uri)
+	{
+		if (!stristr($image_uri, DI::baseUrl() . '/photo/')) {
+			return;
```

Comment:
`return '';` because it is always returning a string. :)

Reply:
done